### DISCUSSION TITLE
Microsoft Visual Studio error window stating - There was a problem adding your Github account

### DISCUSSION THREAD

**Discussion by AnjaniKSingh** [community/community#102750] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

When i add account from visual studio, new browser opens as expected and display authorization was successful. However, i see a pop-up immediately In visual studio stating mentioned error. Please help me troubleshoot. I have Copilot extension along with chat installed in my VS.

**Comment by KowyXYZ** [comment]
Did you try with another account?

**Reply by AnjaniKSingh** [reply]
No, this is my account from my organization.

**Comment by mw-xyz** [comment]
Check with your organisation if they allow access to GitHub copilot. Some don't because of privacy concerns. Let me know if this helps

**Reply by AnjaniKSingh** [reply]
They have asked to start using co-pilot and it's working for few team members.

**Reply by ayim** [reply, marked as answer]
@AnjaniKSingh please file a feedback ticket via the VS Feedback mechanism so that we can take a closer look at your version info and relevant logs.

![image](https://github.com/community/community/assets/10951869/04d7cc2f-d363-4210-857e-1ecfacea23a1)
